Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionControllerCocoa.mm

```diff
@@ -254,6 +254,57 @@
     }
 }
 
+void WebExtensionController::didCommitLoadForFrame(WebPageProxyIdentifier pageID, WebCore::FrameIdentifier frameID, URL frameURL)
+{
+    WebExtensionContext::EventListenerTypeSet listenerTypes;
+    listenerTypes.add(WebExtensionEventListenerType::WebNavigationOnCommitted);
+    // FIXME: Is there a more precise time to fire the DOMContentLoaded event? This matches Safari's behavior.
```

Comment:
Does not appear to be one at the WebKit level.

Reply:
I'll just get rid of the FIXME.